components/job-orchestration/job_orchestration/executor/query/utils.py

```
@@ -119,17 +119,18 @@ def sigterm_handler(_signo, _stack_frame):
 def update_query_task_metadata(
```

Ruff (0.14.5)

Missing docstring in public function

(D103)